### PR TITLE
supplemental table1

### PR DESCRIPTION
**Pull request by MikeLippincott**
This PR creates supplemental table1 for the manuscript

### CODE REVIEW

File: figures/S1.supplemental_table1/notebooks/Stable1.ipynb [hunk 1]
```diff
@@ -0,0 +1,1093 @@
+{
+ "cells": [
```
Comment (d33bs) [inline figures/S1.supplemental_table1/notebooks/Stable1.ipynb:2]: Consider adding a header cell with a title and short description of what the notebook is for.

File: figures/S1.supplemental_table1/results/2023_Interstellar_Table_S1.csv [hunk 2]
```diff
@@ -0,0 +1,188 @@
+cytokine,apoptosis_healthy_p-adj,apoptosis_pyroptosis_p-adj,healthy_pyroptosis_p-adj,none_significant,apoptotic,pyroptotic,healthy,all_significant,putative_function
+AITRL (GITR Ligand),0.8458,0.5658,0.5388,none_significant, , ,, ,Not Annotated
```
Comment (d33bs) [inline figures/S1.supplemental_table1/results/2023_Interstellar_Table_S1.csv:2]: I noticed a pattern of " " spaces vs "" empty values. Would this be important to note or avoid somewhere? Feel free to ignore if this is unhelpful.

File: figures/S1.supplemental_table1/results/consensus_cytokine_results_cleaned.md
Comment (d33bs) [inline figures/S1.supplemental_table1/results/consensus_cytokine_results_cleaned.md:1]: How is this delivered to the audience? Depending on what's helpful, consider exporting directly to a format or visual that can be embedded where it's needed. I'm used to markdown not being compatible with submission so my presumption here is that some further translation might be necessary. If this is displayed on the web somehow, consider using something like [`itables`](https://github.com/mwouts/itables) (or a similar alternative) to help users sift through the data.

**Review by d33bs — APPROVED** [review]
Nice job! I left a few comments, LGTM! Please don't hesitate to let me know if you have any questions.

File: figures/S1.supplemental_table1/notebooks/Stable1.ipynb (hunk 1 above)
Comment (MikeLippincott) [reply figures/S1.supplemental_table1/notebooks/Stable1.ipynb:2]: added!

**Pull request merged**